src/main/java/seedu/address/model/policy/PolicySet.java

```diff
@@ -240,6 +240,21 @@ public boolean retainAll(Collection<?> collection) {
         throw new UnsupportedOperationException("PolicySet does not support the retainAll method.");
     }
 
+    /**
+     * Retrieves all the policies stored in this PolicySet as a list.
+     *
+     * @return a List of all policies in the set.
+     */
+    public List<Policy> getAllPolicies() {
+        List<Policy> policyList = new ArrayList<>();
+        for (Policy policy : policies) {
+            if (policy != null) {
+                policyList.add(policy);
+            }
+        }
+        return policyList;
+    }
```

Comment:
I don't think need this function as commented in AddressBook.java. If there is a need, should use the toArray instead